src/test/locationFactory.test.ts

```diff
@@ -10,7 +10,7 @@ import { LocationFactory } from "../factories/locationFactory";
 
 suite("parseRange", () => {
     test("Undefined range", async () => {
-        const region: sarif.Region | undefined = undefined;
+        const region: sarif.Region = {};
```

Comment:
Parse range does not take undefined. TSLint is disabled for test code, which is why this wasn't caught. We should consider enabling tslint for test code.